.lintstagedrc.json

```diff
@@ -1,5 +1,5 @@
 {
 	"*.js": ["prettier --write"],
-	"*.ts": ["prettier --write", "eslint"],
+	"*.ts": ["prettier --write", "eslint --rulesdir ./tools/eslint-config-web3-base/rules"],
```

Comment:
1. It does not seems to reference a package directory like this, we may move the `eslint-config-web3-base` as standalone package in future. 
2. There must be away to export rules and use those in the `.eslintrc.js` file.